File: curvv/archive/fetch-env-var/getToken.java
public class Main {
    public static void main(String[] args) {
        String value = System.getenv("githubAPI");

        if (value != null) {
            System.out.println("The value of 'githubAPI' is: " + value);
        } else {
            System.out.println("The environment variable 'githubAPI' is not set.");
        }
    }
}

File: curvv/archive/fetch-env-var/getToken.cs
using System;

class Program
{
    static void Main()
    {
        string value = Environment.GetEnvironmentVariable("keyToAll");

        if (!string.IsNullOrEmpty(value))
        {
            Console.WriteLine($"The value of 'keyToAll' is: {value}");
        }
        else
        {
            Console.WriteLine("The environment variable 'keyToAll' is not set.");
        }
    }
}
